zipline/assets/asset_writer.py

```diff
@@ -563,3 +560,9 @@ def _load_data(self, equities, futures, exchanges, root_symbols):
             exchanges=exchanges_output,
             root_symbols=root_symbols_output,
         )
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *exc_info):
+        pass  # Shouldn't dispose of the engine explicitly
```

Comment:
Functionally, no. 

But for the code as a whole, I think it makes sense for all of the writer objects to be context managers as they control system resources. 
This asset writer is a special case because it's querying system doesn't need to be cleaned up explicitly.
If someone were to change the asset writer's underlying system for performance or to remove redundancies, it would be easy to add the exit method if needed.